json-rpc/src/tests/mock_db.rs

```diff
@@ -235,4 +237,29 @@ impl DbReader for MockLibraDB {
     fn get_latest_state_root(&self) -> Result<(u64, HashValue)> {
         unimplemented!()
     }
+
+    fn update_to_latest_ledger(
```

Comment:
I wish there's something like gmock in rust...

Reply:
how would gmock do in this case?